wgsl/index.bs

```diff
@@ -5134,25 +5134,25 @@ Reads a single texel from a texture without sampling or filtering.
-textureLoad(t : texture_3d<T>,                    coords : vec3<u32>)                                                     -> vec4<T>
-textureLoad(t : texture_3d<T>,                    coords : vec3<u32>,                    level : u32)                     -> vec4<T>
-textureLoad(t : texture_multisampled_2d<T>,       coords : vec2<u32>,                                 sample_index : u32) -> vec4<T>
-textureLoad(t : texture_multisampled_2d_array<T>, coords : vec2<u32>, array_index : u32,              sample_index : u32) -> vec4<T>
-textureLoad(t : texture_depth_2d,                 coords : vec2<u32>)                                                     -> f32
-textureLoad(t : texture_depth_2d,                 coords : vec2<u32>,                    level : u32)                     -> f32
-textureLoad(t : texture_depth_2d_array,           coords : vec2<u32>, array_index : u32)                                  -> f32
-textureLoad(t : texture_depth_2d_array,           coords : vec2<u32>, array_index : u32, level : u32)                     -> f32
-textureLoad(t : texture_storage_ro_1d<F>,         coords : u32)                                                           -> vec4<T>
-textureLoad(t : texture_storage_ro_1d_array<F>,   coords : u32,       array_index : u32)                                  -> vec4<T>
-textureLoad(t : texture_storage_ro_2d<F>,         coords : vec2<u32>)                                                     -> vec4<T>
-textureLoad(t : texture_storage_ro_2d_array<F>,   coords : vec2<u32>, array_index : u32)                                  -> vec4<T>
-textureLoad(t : texture_storage_ro_3d<F>,         coords : vec3<u32>)                                                     -> vec4<T>
+textureLoad(t : texture_1d<T>,                    coords : i32)                                                           -> vec4<T>
+textureLoad(t : texture_1d_array<T>,              coords : i32,       array_index : i32)                                  -> vec4<T>
+textureLoad(t : texture_2d<T>,                    coords : vec2<i32>)                                                     -> vec4<T>
+textureLoad(t : texture_2d<T>,                    coords : vec2<i32>,                    level : i32)                     -> vec4<T>
+textureLoad(t : texture_2d_array<T>,              coords : vec2<i32>, array_index : i32)                                  -> vec4<T>
+textureLoad(t : texture_2d_array<T>,              coords : vec2<i32>, array_index : i32, level : i32)                     -> vec4<T>
+textureLoad(t : texture_3d<T>,                    coords : vec3<i32>)                                                     -> vec4<T>
+textureLoad(t : texture_3d<T>,                    coords : vec3<i32>,                    level : i32)                     -> vec4<T>
+textureLoad(t : texture_multisampled_2d<T>,       coords : vec2<i32>,                                 sample_index : i32) -> vec4<T>
```

Comment:
Updated `sample_index` to also be `i32` as this was the only `u32` remaining. Debatable.

Reply:
Now that all other texture parameters are signed, I don't think this is that odd anymore.